Source/WebKit/Platform/IPC/WorkQueueMessageReceiver.h

```diff
@@ -26,14 +26,20 @@
 #pragma once
 
 #include "MessageReceiver.h"
-#include <wtf/ThreadSafeRefCounted.h>
+#include <wtf/ThreadSafeWeakPtr.h>
 
 namespace IPC {
 
-class WorkQueueMessageReceiver : public MessageReceiver, public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<WorkQueueMessageReceiver> {
+class WorkQueueMessageReceiverBase : public MessageReceiver {
+protected:
+    WorkQueueMessageReceiverBase() = default;
+};
+
+template<WTF::DestructionThread destructionThread>
```

Comment:
Can we give this a default value?  Clearly Any is the most common value.

Reply:
I do not want to. I believe call sites should have to think about it and make the decision explicitly instead of having quiet threading bugs.

Also, I do not guarantee that the ones that remain as Any in this PR are truly safe to destroy on any thread. I only converted the obvious ones.